drivers/adc/adc_mcux_lpadc.c

```diff
@@ -93,6 +190,12 @@ static int mcux_lpadc_channel_setup(const struct device *dev,
 
 	LPADC_GetDefaultConvCommandConfig(cmd);
 
+	/* Configure LPADC acquisition time. */
+	if (mcux_lpadc_acquisition_time_setup(dev, channel_cfg->acquisition_time, cmd)) {
+		LOG_ERR(" LPADC acquisition time setting failed");
```

Comment:
extra space in the beginning not needed

Reply:
thanks for your reminder, I will remove the space.